__fixtures__/cli/disallowed.js

```diff
@@ -0,0 +1 @@
+const unused = 1;
```

Comment:
Should this be moved to the base disallowed fixture file?  I don't think we want unused variables pretty much anywhere, right?